openedx/core/djangoapps/external_user_ids/tests/test_models.py

```diff
@@ -0,0 +1,17 @@
+"""
+Tests for external id models.
+"""
+from django.test import TestCase
+
+from student.tests.factories import UserFactory
+from ..models import ExternalId
+
+
+class TestModels(TestCase):
```

Comment:
Please add the annotation here to skip non-lms tests and you can avoid adding the app to Studio if we do not need it.